src/schedule/ScheduleInfoQuery.js

```diff
@@ -206,6 +212,23 @@ export default class ScheduleInfoQuery extends Query {
 
         return `ScheduleInfoQuery:${timestamp.toString()}`;
     }
+
```

Comment:
I think you need to update `[to|from]Protobuf()` methods to support this new field.

Reply:
updated to include field.